quickjs.c

```diff
@@ -11500,7 +11471,7 @@ static JSValue JS_ToQuotedString(JSContext *ctx, JSValue val1)
         default:
             if (c < 32 || is_surrogate(c)) {
                 snprintf(buf, sizeof(buf), "\\u%04x", c);
-                if (string_buffer_puts8(b, buf))
+                if (string_buffer_write8(b, (uint8_t*)buf, 6))
```

Comment:
Why this change? Isn't bug a string?

Reply:
`buf` contains a string, but we know the length so no need for extra `put8` to perform `strlen`.
`string_buffer_write8` should take a `const void *` so we could drop the cast.